src/30_Advanced/Autosuggest.html

```diff
@@ -12,22 +12,25 @@
 -->
 <!--
 ## Setup
-
-First, the markup declares the page as AMP, contains the AMP boilerplate, and
-includes the required AMP extensions.
 -->
+<!-- -->
```

Comment:
I believe if you move `<-- ## setup -->` inside and just before the `<!-- First we include...` you won't need this extra `<!-- -->` to exclude the boiler plate.

Reply:
Thanks, that's a good suggestion!